src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/ManageApprenticesTests/WhenDownloadingApprentices.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.Commitments.Shared.Interfaces;
+using SFA.DAS.ProviderCommitments.Services;
+using SFA.DAS.ProviderCommitments.Web.Controllers;
+
+namespace SFA.DAS.ProviderCommitments.Web.UnitTests.Controllers.ManageApprenticesTests
+{
+    public class WhenDownloadingApprentices
+    {
+        [Test]
+        public async Task ThenTheFileNameIsSetCorrectly()
+        {
+            //Arrange
+            var expected = $"{"Manageyourapprentices"}_{DateTime.Now:yyyyMMddhhmmss}.csv";
+            var controller = new ManageApprenticesController(Mock.Of<ICommitmentsService>(), Mock.Of<ICreateCsvService>());
+
+            //Act
+            var actual = await controller.Download(1);
+
+            var actualFileResult = actual as FileContentResult;
+
+            //Assert
+            Assert.AreEqual(expected, actualFileResult.FileDownloadName);
+        }
```

Comment:
another test is that the payload matches what was returned from the csv service. probably a byte[], but you'll _probably_ be able to assert some kind of equality.